script/aws/download_work_of_participants.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+USER_NAME="intern-user"
+
+usage(){
+  cat<<EOF
+Usage:
+  $(basename "$0") <path to ssh key for ${USER_NAME}>
+EOF
+}
+
+user_key_path="$1"
+
+if [ $# -lt 1 ]; then
+  usage
+  exit
+fi
+
+REMOTE_TARGET_DIR="/home/${USER_NAME}/IoTIntern"
+now=$(gdate +%Y%m%d%H%M%S)
```

Comment:
```suggestion
now=$(date +%Y%m%d%H%M%S)
```

ここは、 `date` で良いように思いました。

手元の mac で bsd 版 date を叩いたとしても、gnu 版と挙動が変わらないことを確認しました。

```
$ /bin/date +%Y%m%d%H%M%S
20210908145109
```

mac では GNU 版は gdate で入っていると思いますが、このコマンド名は他の os において必ずしも当てはまらないと思ったので。

Reply:
確かに、gdate でなくて問題なかったです。ありがとうございます!